src/openforms/appointments/contrib/jcc/constants.py

```diff
@@ -0,0 +1,215 @@
+from django.db.models import TextChoices
+from django.utils.translation import gettext_lazy as _
+
+from openforms.formio.typing import Component, RadioComponent
+
+
+class CustomerFields(TextChoices):
+    """
+    Enum of possible customer field names offered by JCC.
+
+    Documentation reference 4.25 GetRequiredClientFields
+
+    .. note:: The field 'last name' is always required and included here for
+       completeness sake, but never returned from the service.
+
+    For details about the meaning of fields, see 3.1 AppointmentDetailsType in the
+    documentation - that's what the fields map to (with the ``client`` prefix).
+    """
+
+    last_name = "LastName", _("Last name")
+    """
+    Customer last name, max length 128
+    """
+
+    address = "Address", _("Address")
+    """
+    Street name and number - max length 64.
+    """
+
+    birthday = "Birthday", _("Birthday")
+    """
+    Customer date of birth. May not be in the future (assuming this is ``clientDateOfBirth``).
+    """
+
+    city = "City", _("City")
+    """
+    Name of city where customer lives, max length 80.
+    """
+
+    email = "Email", _("Email address")
+    """
+    Contact email address, max length 254 and must be a valid email.
+    """
+
+    initials = "Initials", _("Initials")
+    """
+    Initials (or first name!) of the customer. Max length 128.
+    """
+
+    sex = "Sex", _("Sex")
+    """
+    Gender of the customer, either ``M`` or ``F``.
```

Comment:
I strongly dislike that this field is even an option AND the imposed limit of binary genders. In the context of contact details, this is most likely used to address people (with Mr. or Mrs.), so semantically they care about the gender the person identifies with and not their physical sex. So IMO gender is the more correct phrasing, but we unfortunately can't make it more inclusive because the possible values are binary/limited by JCC.